example_step_workflow/steps/plot/plot.py

```diff
@@ -94,6 +110,33 @@ def run(
         plot_save_path = plot_dir / f"plot.png"
         plt.savefig(plot_save_path, format="png")
 
+        # Plot the vectors in a fancy way
+        plot_matrix = plot_matrix[plot_matrix[:, m - 1].argsort()]
+        max_pm = np.amax(plot_matrix)
+
+        # First make matrix
+        ax = plt.axes()
+        cmap = plt.cm.get_cmap("gnuplot")
+        for i in range(n):
+            y = plot_matrix[i, :]
+            max_y = np.amax(y)
+
+            # Fancy plotting
+            x = np.arange(m)
+            lc = cmap(max_y / max_pm)
+            gradient_fill(x, y, lc)
+
+        # set axes limits
+        plt.xlim(1, m)
+        plt.ylim(0, np.amax(plot_matrix))
+
+        # Configure manifest dataframe for storage tracking
+        self.manifest = pd.DataFrame(index=range(1), columns=["filepath"])
+
+        # Configure save path and save
+        plot_save_path = plot_dir / f"plot_fancy.png"
+        plt.savefig(plot_save_path, format="png")
```

Comment:
I think you are only saving the `plot_fancy.png` to the manifest. I don't think you are saving _both_ files. But since you are planning on splitting this up into two steps I will allow it.